Sources/Basic/FSProxy.swift

```diff
@@ -268,19 +268,19 @@ private class LocalFS: FSProxy {
 //
 // FIXME: This class does not yet support concurrent mutation safely.
 public class PseudoFS: FSProxy {
-    private class Node {
+    fileprivate class Node {
```

Comment:
Huh, this seems unfortunate for exactly this use case. Declaring it as fileprivate *looks* wrong... is it possible this indicates a problem in the proposal?